apps/frontend-pwa/src/lib/getParticipantToken.ts

```diff
@@ -23,13 +23,18 @@ export default async function getParticipantToken({
   let participantToken: string | undefined | null =
     cookies['participant_token'] ?? query.participantToken
 
-  // TODO: reintroduce this after issues with participation creation are in the past
-  // if (participantToken) {
-  //   return {
-  //     participantToken,
-  //     cookiesAvailable: !!cookies['participant_token'],
-  //   }
-  // }
+  // TODO: only check for existing participantToken once participation issues with LTI are resolved
+  if (
+    participantToken &&
+    !cookies['lti-token'] &&
+    !query.jwt &&
+    !(req.method === 'POST')
```

Comment:
@rschlaefli Do you think we can already to back to only checking whether a participationToken exists at this point? The current logic should avoid that the early return is triggered if any of the LTI workflows would be followed afterward and works for students that are normally logged in through the pwa application.

Reply:
yes, we should be able to switch back now